resources/assets/helpers.js

```diff
@@ -42,7 +52,19 @@ export function extractPostsFromSignups(signups) {
 
 export function getEditedImageUrl(photoProp) {
   const edited_file_name = `edited_${photoProp.id}.jpeg`;
-  const url_parts = photoProp['url'].split("/");
+  var url_parts;
+
+  // Sometimes we get the url right on the post and sometimes it is nested under
+  // media (in cases where it goes through the PostTransformer), so handle both cases
+  if ('url' in photoProp) {
+    url_parts = photoProp['url'].split("/");
```

Comment:
Can we pull the rest of this condition (the code below the if/else) up into the conditional?